cachito/web/models.py

```diff
@@ -325,7 +326,7 @@ class Request(db.Model):
         db.Integer, db.ForeignKey("request_state.id"), index=True, unique=True
     )
     state = db.relationship("RequestState", foreign_keys=[request_state_id])
-    request_dependencies = db.relationship("RequestDependency")
+    # request_dependencies = db.relationship("RequestDependency")
```

Comment:
Did you mean to do this?
```suggestion
    request_dependencies = db.relationship("RequestDependency", backref="request")
```